app/views/posts/index.erb

```diff
@@ -1,112 +1,6 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <style>
-      .user-details {
-        display: flex;
-        align-items: center;
-        margin-bottom: 20px;
-      }
-
-      .user-details .image {
-        margin-right: 10px;
-      }
-
-      .user-details img {
-        width: 100px;
-        height: 100px;
-        object-fit: cover;
-        border-radius: 50%;
-      }
-
-      .user-details h2 {
-        margin: 0;
-        font-size: 24px;
-        font-weight: bold;
-      }
-
-      .user-details p {
-        margin: 0;
-        color: gray;
-      }
-      /* Styles for posts */
-      .post-list {
-        list-style-type: none;
-        padding: 0;
-      }
-
-      .post-list li {
-        margin-bottom: 20px;
-      }
-
-      .post-list h3 {
-        margin: 0;
-        font-size: 20px;
-        font-weight: bold;
-      }
-
-      .post-list p {
-        margin: 0;
-      }
-
-      .post-meta {
-        color: gray;
-      }
-
-      .comments-container {
-        margin-top: 10px;
-        padding: 10px;
-        border: 1px solid #ccc;
-        border-radius: 4px;
-        background-color: #f9f9f9;
-      }
-
-      /* Button style */
-      .show-comments-button {
-        background-color: #4caf50;
-        color: white;
-        padding: 5px 10px;
-        border: none;
-        border-radius: 4px;
-        cursor: pointer;
-        font-size: 14px;
-        margin-top: 10px;
-      }
-
-      .show-comments-button:hover {
-        background-color: #45a049;
-      }
-    </style>
-  </head>
-  <body>
-    <h1>List of Posts For User <%= @user.id %></h1>
-    <ul class="post-list">
-      <div class="user-details">
-        <div class="image">
-          <img
-            src="https://www.pngfind.com/pngs/m/610-6104451_image-placeholder-png-user-profile-placeholder-image-png.png"
-            alt="user"
-          />
-        </div>
-        <div>
-          <h2><%= @user.name %></h2>
-          <p><%= @user.bio %></p>
-        </div>
-      </div>
-      <% @posts.each do |post| %>
-      <li>
-        <h3><%= post.title %></h3>
-        <p><%= post.text %></p>
-        <div class="post-meta">
-          Likes: <%= post.likes_counter %> | Comments: <%= post.comments_counter
-          %>
-        </div>
-        <div class="comments-container">
-          <!-- Code to display comments goes here -->
-        </div>
-        <button class="show-comments-button">Show comments</button>
-      </li>
-      <% end %>
-    </ul>
-  </body>
-</html>
+<h1>List of Posts For User <%= @user.id %></h1>
+<%= render partial: 'users/user', locals:{user:@user}%> 
+<% @posts.each do |post|%> 
```

Comment:
- [ ] Here, you are supposed to display the most recent comments using the method you created in your posts model. It should be `@users.most_recent_comments.each`